src/nvim/ui_compositor.c

```diff
@@ -275,7 +275,7 @@ ScreenGrid *ui_comp_mouse_focus(int row, int col)
 {
   for (ssize_t i = (ssize_t)kv_size(layers) - 1; i > 0; i--) {
     ScreenGrid *grid = kv_A(layers, i);
-    if (grid->focusable
+    if (grid->mouse
```

Comment:
Wonder if this should be something like
```
grid->mouse != kWinMouseIgnore
```

Reply:
`WinMouseEvent` is only used in `w_config`. In `ScreenGrid` a simple boolean is used. I've renamed that to `mouse_enabled` to avoid confusion.